editors/kate/Jakt.xml

```diff
@@ -0,0 +1,222 @@
+                <keyword attribute="JaktConditional" context="#stay" String="jaktConditional" />
+                
+            </context>
+            
+            <!-- End handling multiple-line string : jaktDoubleQuot -->
+            <context attribute="String" lineEndContext="#stay" name="jaktDoubleQuot">
+                <DetectChar attribute="String" context="#pop" char="&quot;" />
+            </context>
+            
+            <!-- End handling multiple-line string : jaktSingleQuot -->
+            <context attribute="String" lineEndContext="#stay" name="jaktSingleQuot">
+                <DetectChar attribute="String" context="#pop" char="'" />
+            </context>
+            
+            <!-- End handling multiple-line comment : Comment -->
+            <context name="Comment" attribute="Comment" lineEndContext="#stay">
+                <DetectSpaces/>
+                <StringDetect attribute="Comment" context="#pop" String="*/" endRegion="comment"/>
+            </context>
+
+        </contexts>
+        <!-- End Contexts -->
+
+        <itemDatas>
+            <itemData name="NormalText" defStyleNum="dsNormal" />
+            <itemData name="JaktFunction" defStyleNum="dsFunction" />
+            <itemData name="JaktlFunctionLable" defStyleNum="dsRegionMarker" />
+            <itemData name="JaktType" defStyleNum="dsDataType" />
+            <itemData name="JaktWordOperator" defStyleNum="dsKeyword" />
+            <itemData name="JaktKeyword" defStyleNum="dsKeyword" />
+            <itemData name="JaktException" defStyleNum="dsSpecialChar" />
+            <itemData name="JaktVarDecl" defStyleNum="dsBuiltIn" />
+            <itemData name="JaktStructure" defStyleNum="dsKeyword" />
+            <itemData name="JaktVisModifier" defStyleNum="dsKeyword" />
+            <itemData name="JaktConstant" defStyleNum="dsKeyword" />
+            <itemData name="JaktMacro" defStyleNum="dsSpecialChar" />
+            <itemData name="JaktKeyword" defStyleNum="dsKeyword" />
+            <itemData name="JaktBoolean" defStyleNum="dsBuiltIn" />
+            <itemData name="JaktExecution" defStyleNum="dsSpecialChar" />
+            <itemData name="JaktRepeat" defStyleNum="dsKeyword" />
+            <itemData name="JaktConditional" defStyleNum="dsKeyword" />
+            <itemData name="JaktlSymbols" defStyleNum="dsSpecialChar" />
+            <itemData name="String" defStyleNum="dsString" />
+            <itemData name="Symbol" defStyleNum="dsSpecialChar" />
+            <itemData name="Comment" defStyleNum="dsComment" />
+            <itemData name="MultipleComment" defStyleNum="dsComment" />
+        </itemDatas>
+    </highlighting>
+
+    <!-- Start General -->
+    <general>
+        <!-- Comments -->
+        <comments>
+            <comment name="singleLine" start="//" />
+            <comment name="multiLine" start="/*" end="*/" region="CommentFolding" />
```

Comment:
This comment form doesn't actually exist.

Reply:
Yep, I will fix it in next request.